glossary/kontrol-toml.md

```diff
@@ -0,0 +1,165 @@
+# Configuration Options for Kontrol TOML file
+
+The following is a kontrol.toml file with all configuration options set.
+
+```toml
+[build.default]
+foundry-project-root       = '.'
+regen                      = false
+rekompile                  = false
+verbose                    = false
+debug                      = false
+require                    = 'lemmas.k'
+module-import              = 'TestBase:KONTROL-LEMMAS'
+auxiliary-lemmas           = true
+# definition_dir = 
+# spec_module =
+
+[build]
+backend = # K backend to target with compilation.
```

Comment:
I don't think that's allowed in a non-comment line, please consider changing it (and other similar lines) to the following:
```suggestion
# backend = # K backend to target with compilation.
```
Please check if `kontrol build`, `prove`, etc. works with this `kontrol.toml` file.

Reply:
Sorry, I missed these options. I'll check it again.